storageExample.go

```diff
@@ -3,223 +3,229 @@ package main
 
 import (
 	"encoding/base64"
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"math/rand"
 	"os"
 
+	"bytes"
```

Comment:
Nit, move this to the top of the imports list (seems like a recent change in the formatter screws this up).